apps/chat/src/components/Chat/ChangePathDialog.tsx

```diff
@@ -152,6 +174,18 @@ export const ChangePathDialog = ({
 
   const handleAddFolder = useCallback(
     (parentFolderId: string) => {
+      const folderName = getNextDefaultName(
+        t(DEFAULT_FOLDER_NAME),
+        folders,
+        0,
+        false,
+        true,
+      );
+
+      setSelectedFolderId(
+        constructPath(parentFolderId || getConversationRootId(), folderName),
```

Comment:
Should it be just `rootFolderId` instead of `getConversationRootId()`?

Reply:
sure, i'll fix it